packages/cx-templates/doc/README.md

```diff
@@ -50,40 +52,59 @@ As they design templates, Site Builders can choose the following:
 * Where to place Flow Containers, allowing flexible areas in the template.
 * Different variations of Flow Containers that control what components could be added to an area.
 
-## Content Editor
-
-When the content editor creates a new page, it will create a new page using the default template.
-They can go to Page | Template and choose which template they would like to switch to.  (@jones see /styleguide section and you will get the templates button)
-
-## Site Builder
-
-The choice of what templates and setting templates is done within the [cxPage](https://github.com/johnsonandjohnson/Bodiless-JS/tree/main/packages/cx-templates/src/components/Page) Component.
-
-First create a Page token defining what templates & their tokens should apply to a page option and exported as necessary.
-
-````js
-const Default = asFluidToken({
-  ...cxPage.Default,
-  Components: {
-    _default: on(GenericTemplateClean)(cxGenericTemplate.Default),
-    PDP: on(PDPClean)(cxPDP.Default),
-    PLP: on(PLPClean)(cxPLP.Default),
-  },
-});
-
-export default {
-  ...cxPage,
-  Default,
-};
-
-````
-
-Then within the _default.jsx file in the sites/NAMEOFSITE/src/templates it will call use this token.
-
-````js
-const DefaultPage = as(__cxstarter__Page.Default)(Fragment);
-
-export default DefaultPage;
-```
-
-@jones -- i think there is one more step to get the choice of Templates in the Template UI list seen by content editor -- need to fidn that step. :)
+## Content Editor Details
+
+When you create a new page, it will create the new page using the default template.
+
+### Change Template
+
+![Template icon](../assets/PageTemplateIcon.jpg ':size=60')
+
+To switch the template that a page is using:
+
+01. Navigate to the page for which you'd like to change the template.
+01. From the [Toolbar](/ContentEditorUserGuide/#toolbar), click **Page > Template** to open the
+    _Choose a template for this page_ form.  
+    !["Choose a template for this page" form](../assets/PageChooseTemplate.jpg ':size=50%')
+01. Select the radio button of the template you'd like to switch to, and click the checkmark to
+    confirm.
+01. The page template should automatically update.
+
+## Site Builder Details
+
```

Comment:
A Site builder can use [gatsby template](https://www.gatsbyjs.com/docs/conceptual/building-with-components/#page-template-components) within the site and place the created template components directly within the template.

The Vital Design offers an alternative method to using gatsby templates that allows content editor a choice to quickly switch a page between templates.    A generic Page component is placed on the sites' gatsby _default template file and this component offers individual template/layout components to choose from.   Each 'template/layout' component  is a component that renders the pages contents in its  specific requirements.